### PR TITLE
First attempt to commit dll.js Patch 1

### PR DESCRIPTION
**Pull request by thednp**
First stable release of dll.js

### CODE REVIEW

File: files/dll.js/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "dnp_theme"
+github = "https://github.com/thednp/dll.js"
+homepage = "http://themeforest.net/user/dnp_theme"
```
Comment (megawac) [inline files/dll.js/info.ini]: Is this site relevant

Comment (thednp) [inline files/dll.js/info.ini]: Well, it's relevant to the author, should it be to the file?

Comment (megawac) [inline files/dll.js/info.ini]: Should be relevant to the library being uploaded

Comment (thednp) [inline files/dll.js/info.ini]: I see no issue with that, I have no other homepage or demo so far.

**Comment by jimaek** [comment]
The homepage should be the website of the project being submitted, not the author.
If the project doesnt have a website then you can simply put there the github repo

**Pull request closed without merging**